src/application/shell.ts

```diff
@@ -308,6 +308,51 @@ class ApplicationShell extends Widget {
     this._save();
   }
 
+  /*
+   * Activate the next Tab in the active TabBar.
+  */
+  activateNextTab(): void {
+    let current = this._currentTabBar();
+    if (current) {
+      let ci = current.currentIndex;
+      if (ci !== -1) {
+        if (ci < (current.titles.length-1)) {
```

Comment:
While we're being nit-picky, you don't need the extra parens either. The `-` operator has a higher precedence than logical operators: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence